src/graphics/gfx_enter_screen.cpp

```diff
@@ -18,56 +18,133 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <SDL2/SDL_timer.h>
+
 #include "../globals.h"
 #include "../graphics.h"
 #include "../player.h"
+#include "../frame_timer.h"
+#include "../screen_fader.h"
+#include "pge_delay.h"
 
 
-void GameThing()
+static void initPlayers(Player_t tempPlayer[2])
```

Comment:
Can you instead use `maxLocalPlayers` here? Not sure if C arrays support constants here, but the abstract-controls code is trying to allow for >2 players in the future.

Reply:
Good note!

Reply:
Fixed :+1: